lib/role.rb

```diff
@@ -4,11 +4,17 @@
 # convenience methods for testing the role status of users.
 module Role
 
+  def grant_role!(role, facility: nil)
```

Comment:
Do we need this method? It doesn't look like this gets used anywhere, and we have We have `UserRole.grant`, which should probably be doing the valid roles check.